agent/tool-scripts/base-tool

```diff
@@ -579,17 +579,19 @@ install)
 	function check_required_rpm {
 		local _tpn="${1}"
 		local _tpv="${2}"
-		local _installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
-		local rc=${?}
-		if [[ ${rc} != 0 ]]; then
+		local _installed_rpm
+		local _rc
+		_installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
+		_rc=${?} 
+		if [[ ${_rc} != 0 ]]; then
 			if [[ ! -z "${_tpv}" ]]; then
 				_tpv="-${_tpv}"
 			fi
 			printf -- "%s: %s is not installed\n" "${tool}" "${_tpn}${_tpv}" >&2
```

Comment:
Don't know if you're interested, but I think these four lines can be collapsed into one:
```
			printf -- "%s: %s is not installed\n" "${tool}" "${_tpn}${_tpv:+-${_tpv}}" >&2
```